command/alias.go

```diff
@@ -112,3 +111,43 @@ func processArgs(args []string) []string {
 
 	return newArgs
 }
+
+var aliasDeleteCmd = &cobra.Command{
+	Use:     "delete <alias>",
+	Short:   "Delete an alias.",
+	Args:    cobra.ExactArgs(1),
+	Example: "gh alias delete co",
+	RunE:    aliasDelete,
+}
+
+func aliasDelete(cmd *cobra.Command, args []string) error {
+	alias := args[0]
+
+	ctx := contextForCommand(cmd)
+	cfg, err := ctx.Config()
+	if err != nil {
+		return err
+	}
+
+	aliasCfg, err := cfg.Aliases()
+	if err != nil {
+		return err
+	}
+
+	if !aliasCfg.Exists(alias) {
+		return fmt.Errorf("no such alias %s", alias)
+	}
+
+	expansion := aliasCfg.Get(alias)
```

Comment:
I was wondering what the return value would be if you didn't check for existence on line 137. This got me thinking about making the `Get` method work more like the Go map lookup. So the interface would be:

`func Get(alias string) (value string, found bool)`

This isn't something we have to do, but it feels like it might be more standard Go style.

Reply:
that would make sense :+1: